packages/jest-leak-detector/build/index.ts_

```diff
@@ -3,8 +3,6 @@
  *
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
- *
- * @flow
  */
```

Comment:
this should be renamed to `src/index.ts` not `build/index.ts_`

Reply:
yeah, it my mistake. One moment, will cleanup all stuff.

Reply:
@thymikee btw, as far as I understand, I don't need to commit files from `build` directory, right?

Reply:
Yup, all `build`s are ignored and it should stay this way :)

Reply:
Thanks! 👍

Reply:
You'll need to explicitly call `git rm packages/jest-leak-detector/build/index.ts_`

Reply:
Yes, you're right! Thanks again ;)